### PR TITLE
Cohen's d revamp

### PR DESCRIPTION
**Pull request by mattansb**

### CODE REVIEW

File: vignettes/standardize_data.Rmd [hunk 1]
```diff
@@ -50,22 +52,185 @@ Standardization is an important step and extra caution is required in **repeated
...
+```{r, fig.width=7, fig.height=4.5, eval = TRUE, results='markup', fig.align='center', comment=NA, message=FALSE, warning=FALSE}
+library(bayestestR)
+library(see)
+
+data.frame(VarWise = Z_VariableWise$Subjective_Valence,
+           ParWise = Z_ParticipantWise$Subjective_Valence,
+           Full = Z_Full$Subjective_Valence) %>% 
+  estimate_density(method="kernSmooth") %>% 
+  plot() +
+  see::theme_modern()
```
Comment (mattansb) [inline vignettes/standardize_data.Rmd]: @DominiqueMakowski can't get this bit to work.
I have dev version of see, but see no `see:::plot.see_estimate_density_df` method there.

Comment (DominiqueMakowski) [reply vignettes/standardize_data.Rmd]: try with the master `see`? Although having a nice `NEWS`, `see` has sometimes a dev behind master 🤷‍♂

Comment (DominiqueMakowski) [reply vignettes/standardize_data.Rmd]: I think it's related to a recent change that we made with @strengejacke

Comment (strengejacke) [reply vignettes/standardize_data.Rmd]:
``` r
library(easystats)
#> # Attaching packages
#> <U+2714> insight     0.8.2.1   <U+2714> bayestestR  0.5.3  
#> <U+2714> performance 0.4.4.1   <U+2714> parameters  0.6.0.1
#> <U+2714> see         0.4.1.1   <U+2714> effectsize  0.3.0  
#> <U+2714> correlation 0.1.1     <U+2714> modelbased  0.1.2  
#> <U+2714> report      0.1.0
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union

iris %>% 
  estimate_density(method = "kernSmooth") %>% 
  plot() +
  theme_modern()
#> Loading required namespace: KernSmooth
```



``` r

iris %>% 
  group_by(Species) %>% 
  estimate_density(method = "kernSmooth") %>% 
  plot() +
  theme_modern()
```



<sup>Created on 2020-03-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Pull request merged**